test/weak_test.py

```diff
@@ -0,0 +1,470 @@
+from weakref import WeakSet
```

Comment:
I'm not sure if we need this weak_test.py in here?

There seems to be enough tests in sprite_test.py for WeakSprite and such.